Stellar-ledger-entries-limits-v2.x

```diff
@@ -12,21 +13,34 @@ enum StatsOpType
     PAYOUT = 5
 };
 
+// `LimitsV2Entry` represents all the details of limits attached to the particular account or account role or globally in the system
```

Comment:
is used in system configuration to set limits (daily, weekly, montly, annual) for different assets, operations (according to StatsOpType) either globally(with no account role provided), or for account role, or for account.